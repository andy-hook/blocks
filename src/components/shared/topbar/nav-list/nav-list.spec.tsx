import React from "react"
import { renderWithTheme } from "@test-utils"

import NavList from "./nav-list"

describe("<Navigation />", () => {
  it("renders correctly", () => {
    const tree = renderWithTheme("dark", <NavList />)
    expect(tree).toMatchSnapshot()
  })
})
