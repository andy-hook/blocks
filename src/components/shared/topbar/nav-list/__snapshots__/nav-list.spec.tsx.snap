// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Navigation /> renders correctly 1`] = `
<nav
  className="sc-bdVaJa bDWFJH"
>
  <ul
    className="sc-bwzfXH iPhcok"
  >
    <li
      className="sc-htpNat iaJVDJ"
    >
      <a
        className="sc-bxivhb iUcbWs"
        href="/"
      >
        Latest blocks
      </a>
    </li>
    <li
      className="sc-htpNat iaJVDJ"
    >
      <a
        className="sc-ifAKCX kdWyHd"
        href="https://github.com/andy-hook/blocks"
        onClick={[Function]}
        target="_blank"
      >
        View on Github
      </a>
    </li>
  </ul>
</nav>
`;
